schematic/models/validate_manifest.py

```diff
@@ -65,6 +65,30 @@ def get_multiple_types_error(
             error_val = f"Multiple Rules: list not first"
         return ["NA", error_col, error_message, error_val]
 
+    def check_max_rule_num(
+        self, validation_rules: str, col: pd.core.series.Series, errors: list[list[str]]
```

Comment:
I think validation rules should be a `list[str]` given how its being copied and having items removed from it

Reply:
Youre right!